src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { formatResults } from './format-results';
+
+// special TS syntax: https://github.com/microsoft/TypeScript/issues/2719
+export = formatResults;
```

Comment:
This is a problem related to interop conflicts with CommonJS and ES6 modules. ESLint expects the formatter to be a function and not an object. If I do:

```
export default formatResults;
```

it complains:

```
Formatter must be a function, but got a object.
```

This is because by default TS compiles to ES6 modules. ESLint would have to require it this way `require('path/to/formatter.js').default` but instead it just `require('path/to/formatter.js')`. To make TS to compile this to a function we do `export = formatResults`. This is described in the link provided in the comment above.

Reply:
Why wouldn't `export { default as formatResults } from './format-results';` work?

Reply:
We don't have control over ESLint code and how they require. Their require expect a function (CommonJS) and not an object with default or other exported members (ES6 modules). We need to `export = formatResults;` for this one.

Reply:
OIC, yes this is why - you're doing this so this is readable by eslint. Got it.